src/overlays/actors/ovl_Bg_Spot11_Oasis/z_bg_spot11_oasis.c

```diff
@@ -6,6 +6,22 @@
 
 #include "z_bg_spot11_oasis.h"
 #include "overlays/actors/ovl_En_Elf/z_en_elf.h"
+
+#include "libc64/qrand.h"
+#include "gfx.h"
+#include "gfx_setupdl.h"
+#include "ichain.h"
+#include "one_point_cutscene.h"
+#include "sfx.h"
+#include "sys_math3d.h"
+#include "sys_matrix.h"
+#include "z_lib.h"
+#include "z64effect.h"
+#include "z64play.h"
+#include "z64player.h"
+
+#include "global.h"
```

Comment:
I'd prefer to see headers being split prior to fixing includes so we can avoid lingering global.h includes, like with `CutsceneFlags_` functions here that are still in functions.h

but it's ok